Cargo.toml

```diff
@@ -15,7 +15,7 @@ path = "bin/ch2rs.rs"
 
 [dependencies]
 anyhow = "1.0.40"
-clickhouse = "0.12.1"
+clickhouse = { version = "0.12.2", features = ["rustls-tls"] }
```

Comment:
this feature should be guided by library consumer

Reply:
would be nice to pick `rustls` as a default but leave the user an option to change to `native` if they desire. Makes using the library much easier.

Reply:
@paymog, @serprex, maybe like this?

```
[features]
default = ["rustls-tls"]
rustls-tls = ["clickhouse/rustls-tls"]
native-tls = ["clickhouse/native-tls"]
```

Reply:
Ok, added.

> `rustls-tls-aws`

@serprex, we need to add it to this crate after 0.13.0 (not available atm)